nanopub/nanopub.py

```diff
@@ -66,6 +67,9 @@ def from_assertion(cls, assertion_rdf, uri=DEFAULT_URI, introduces_concept=None,
 
         """
 
+        if not attributed_to and not nanopub_author:
+            nanopub_author = rdflib.URIRef(profile.get_orcid())
+
```

Comment:
```suggestion
        if attributed_to is None:
            attributed_to is rdflib.URIRef(profile.get_orcid())
        if nanopub_author is None:
            nanopub_author = rdflib.URIRef(profile.get_orcid())
```
I think for both we would like to fall back to the ORCID iD.

Reply:
The nanopub_author is for the whole nanopub, the attributed_to is for the assertion only.

Reply:
Oh ok so it is actually more reasonable that the user is the author of the nanopub? Then I would vote for just defaulting nanopub_author to the user. We don't know whether the assertion is something the author discovered/made/invented/whatever or that he/she has just found something from a paper and wants to publish it as nanopub

Reply:
This is how I see it:

1. By default, the user in the profile is both the nanopub author and the assertion author.
2. If someone else is the nanopub author the user should provide the nanopub_author himself.
3. If someone else is the assertion author the user should provide the attributed_to himself.

The introduction of `use_profile` only complicates things and doesn't add anything IMO. You could argue that you want to protect users from accidentally publishing something attributing to themselves. To prevent that you could indeed have a `use_profile` argument, but it should by default be False and if`use_profile` is False we should raise errors when `attributed_to` argument is empty. Let's discuss!

Reply:
I implemented this.